### DISCUSSION TITLE
Tiddlyhost seems a little slow

### DISCUSSION THREAD

**Discussion by kookma** [simonbaird/tiddlyhost#10] [Q&A]
Many thanks @simonbaird for this great Tiddlyspot alternative.
I am experimenting with tiddlyhost. For me it is slow! Is there any reason for this? Or this is because tiddlyhost is not finalized yet.

**Comment by simonbaird** [comment]
It's running on a low spec Dreamhost VM located in the US, but it's reasonably responsive for me. Where are you accessing it from? How slow is it?

**Reply by kookma** [reply]
Well, I am accessing it from Ir. It is pretty slow on loading/saving but then it works fine!

**Comment by amreus** [comment]
Not sure how difficult it may be but what if there was a toggle to switch between saving local and saving back to TH? So the workflow would be visit your wiki at TH, toggle to saving locally, make a lot of changes, toggle to saving to TH for the final save.
